### PR TITLE
Update Phoenix Contact status

### PR DESCRIPTION
**Pull request by ljaenicke**
Cloud services were either not affected or are fixed now. Affected services have been checked and no exploit was observed.

### CODE REVIEW

File: software/software_list_p.md [hunk 1]
```diff
@@ -98,9 +98,9 @@
...
 |Philips|VuePACS|12.2.8| |Vulnerable| | | |[source](https://www.philips.com/a-w/security/security-advisories.html)|
-|Phoenix Contact|Cloud Services| | |Vulnerable| | |Remediations are being implemented|[source](https://dam-mdc.phoenixcontact.com/asset/156443151564/1a0f6db6bbc86540bfe4f05fd65877f4/Vulnerability_Statement_Log4J_20211215.pdf)|
-|Phoenix Contact|Physical products containing firmware| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://dam-mdc.phoenixcontact.com/asset/156443151564/1a0f6db6bbc86540bfe4f05fd65877f4/Vulnerability_Statement_Log4J_20211215.pdf)|
-|Phoenix Contact|Software products| |Not vuln|Not vuln|Not vuln|Not vuln| |[source](https://dam-mdc.phoenixcontact.com/asset/156443151564/1a0f6db6bbc86540bfe4f05fd65877f4/Vulnerability_Statement_Log4J_20211215.pdf)|
+|Phoenix Contact|Cloud Services| |Fix|Fix|Fix|Fix|Cloud Services were either not vulnerable or are completely fixed. No exploits observed.|[source](https://dam-mdc.phoenixcontact.com/asset/156443151564/1a0f6db6bbc86540bfe4f05fd65877f4/Vulnerability.pdf)|
```
Comment (tschmidtb51) [inline software/software_list_p.md:101]: @ljaenicke Did any of the cloud services use Log4j 1.x? If not, I suggest:
```suggestion
|Phoenix Contact|Cloud Services| |Not vuln|Fix|Fix|Fix|Cloud Services were either not vulnerable or are completely fixed. No exploits observed.|[source](https://dam-mdc.phoenixcontact.com/asset/156443151564/1a0f6db6bbc86540bfe4f05fd65877f4/Vulnerability.pdf)|

**Review by tschmidtb51 — COMMENTED** [review]
@ljaenicke Thank you for your contribution.

Can you please have a look at my comment? I just want to clarify that before merging. Thanks!

File: software/software_list_p.md (hunk 1 above)
Comment (ljaenicke) [reply software/software_list_p.md:101]: @tschmidtb51  Not all our internal reports had this level of detail. I would recommend to just keep as is. From my perspective the "No exploits" is the more important part of the message :-).

Comment (tschmidtb51) [reply software/software_list_p.md:101]: I was just spotting the inconsistency as the advisory did not mention CVE-2021-4104... In such a case we usually assume that it was 'Not vuln' against CVE-2021-4104 in the first place...

Comment (ljaenicke) [reply software/software_list_p.md:101]: Which may need another round of internal reviews and may take some days next week...

Comment (tschmidtb51) [reply software/software_list_p.md:101]: Just for the record: it also does not mention CVE-2021-45105 but as CVE-2021-44832 fixed version include also the fix for CVE-2021-45105 this is okay for us.

Comment (tschmidtb51) [reply software/software_list_p.md:101]: @ljaenicke There are two options here which are backed from the current state of the advisory:
1. We state CVE-2021-4104 as `Not vuln`.
2. We leave CVE-2021-4104 empty.

As you wrote `Fix` I would tend to the first one - but that's your call to make.

Comment (tschmidtb51) [reply software/software_list_p.md:101]: Both ways I'm happy to merge it tonight.

Comment (ljaenicke) [reply software/software_list_p.md]: I have just pushed the "empty" approach.

**Review by tschmidtb51 — APPROVED** [review]
@ljaenicke Thank you for your contribution.

**Pull request merged**